common/common-rest/src/main/java/org/apache/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -268,16 +268,8 @@ protected void sendResponseQuietly(Response response) {
 
   @SuppressWarnings("deprecation")
   protected void sendResponse(Response response) {
-    if (response.getHeaders().getHeaderMap() != null) {
-      for (Entry<String, List<Object>> entry : response.getHeaders().getHeaderMap().entrySet()) {
-        for (Object value : entry.getValue()) {
-          if (!entry.getKey().equalsIgnoreCase(HttpHeaders.CONTENT_LENGTH)
-              && !entry.getKey().equalsIgnoreCase("Transfer-Encoding")) {
-            responseEx.addHeader(entry.getKey(), String.valueOf(value));
-          }
-        }
-      }
-    }
+    RestServerCodecFilter.encodeHeader(response.getHeaders().getHeaderMap(), responseEx);
```

Comment:
`encodeHeader` name is not good. It's do not do any encoding.

Reply:
done